packages/app/src/cli/services/dev/extension/utilities.ts

```diff
@@ -51,7 +51,7 @@ export function getExtensionPointTargetSurface(extensionPointTarget: string) {
     }
 
     // Covers POS UI extensions (future)
-    case 'retail': {
+    case 'pos': {
```

Comment:
I guess this was implemented before the final decision was made, but we ended up going with pos a while ago. This is necessary in order to get the correct surface on the extension target object (which is then used to generate the QR code)